## csr1

PRE CONFIG:

snmp-server community networktocode RW
snmp-server community towerbridge RO
snmp-server community NTCRO123 RO
snmp-server community SGLONDON RO
snmp-server community SGLONDON123 RW
snmp-server community SGLON RO
snmp-server community NTCRW RW
snmp-server community NTCRW4123 RW
snmp-server community NTCCCC001 RO
snmp-server community sgsgsg RO
snmp-server community NTC-LAM-LAM RO

POST CONFIG:

snmp-server community networktocode RW
snmp-server community towerbridge RO
snmp-server community NTCRO123 RO
snmp-server community SGLONDON RO
snmp-server community SGLONDON123 RW
snmp-server community SGLON RO
snmp-server community NTCRW RW
snmp-server community NTCRW4123 RW
snmp-server community NTCCCC001 RO
snmp-server community sgsgsg RO
snmp-server community NTC-LAM-LAM RO
snmp-server community LONDOAAA RO

DIFF:

--- 
+++ 
@@ -9,3 +9,4 @@
 snmp-server community sgsgsg RO
 snmp-server community NTC-LAM-LAM RO
+snmp-server community LONDOAAA RO
